internal/manifest/testdata/oci1.index.zstd-selection.json

```diff
@@ -0,0 +1,66 @@
+{
+    "schemaVersion": 2,
+    "mediaType": "application/vnd.oci.image.index.v1+json",
+    "manifests": [
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74a9522510",
+            "size": 758,
+            "platform": {
+                "architecture": "amd64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:ac9bbf71dacfad2bfbe48f7d46ca279b1b3b351a6c814db760030015167405b8",
+            "size": 772,
+            "annotations": {
+                "io.github.containers.compression.zstd": "true"
+            },
+            "platform": {
+                "architecture": "amd64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74a9522511",
+            "size": 758,
+            "platform": {
+                "architecture": "arm64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74z9522511",
+            "size": 758,
+            "platform": {
+                "architecture": "arm64",
+                "os": "linux"
+            }
+        },
+        {
+            "mediaType": "application/vnd.oci.image.manifest.v1+json",
+            "digest": "sha256:88cb27f194cce3b06994b4af2dd8a5642f4f294b71bda5b74a3fbb74z9522511",
```

Comment:
These two digests are _exactly_ the same.

1. That’s unrealistic, and
2. Unless this is explicitly testing for duplicate digest handling (which is not relevant to our code), this makes for an unhelpful test case because we don’t know which one matched.